broadcast/routes/auth.py

```diff
@@ -195,7 +195,7 @@ def get(self, key):
 
 
 class ResetPassword(ConfirmationMixin, CSRFMixin, LoginOnSuccessMixin,
-                    ActionXHRPartialFormRoute):
+                    NextPathMixin, ActionXHRPartialFormRoute):
```

Comment:
Instead of using NextPathMixin, please try `success_url = ('auth:login', {})` property.